js/two-factor.js

```diff
@@ -0,0 +1,35 @@
+/* global ajaxurl */
+jQuery(document).ready(function ($) {
+
+	$('.delete').on('click', function( e ) {
+		e.preventDefault();
+		var link = $(this).find('a');
+		var data = {
+			'_nonce_delete_application_password': $(link).data('nonce'),
+			'delete_application_password':        $(link).data('deleteApplicationPassword'),
+			'action':                             $(link).data('action')
+		};
+
+		$.post(ajaxurl, data, function(r) {
+			if( true === r.success ) {
+				$('tr[data-slug="' + $(link).data('deleteApplicationPassword') + '"]').remove();
+			}
+		});
+	});
+
+	$('#do_new_application_password').on('click', function( e ) {
+		e.preventDefault();
+		var app_name = $('input[name="new_application_password_name"]').val();
```

Comment:
We should only have one `var` at the very beginning and before any other code. `e.preventDefault();` can be moved below. A separate, and personal preference is to use `event` and not `e` for clarity. Though I can live without it.